packages/x-data-grid-pro/src/hooks/features/infiniteLoader/useGridInfiniteLoader.tsx

```diff
@@ -126,4 +139,13 @@ export const useGridInfiniteLoader = (
 
   useGridApiMethod(apiRef, infiniteLoaderPrivateApi, 'private');
   useGridApiOptionHandler(apiRef, 'rowsScrollEnd', props.onRowsScrollEnd);
+
+  /**
+   * EFFECTS
+   */
```

Comment:
Nitpick: We discarded using JSDoc style comments to identify code sections as the IDE misinterprets it as the documentation of the subsequent method.
```suggestion
```